deps/exokit-bindings/videocontext/include/VideoMode.h

```diff
@@ -0,0 +1,43 @@
+#ifndef _HTML_VIDEO_MODE_ELEMENT_H_
```

Comment:
What's a "video mode element"?

Reply:
@shawwn if this is not an element (which is a well-defined term) then the include guard should not refer to it as such.